build-support/bin/go/mac/10.11/1.9.4/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
It looks like all of the OSX scripts can be symlinks to one copy, since golang doesn't have multiple builds?

Reply:
But... maybe we don't care. Still trying to work out a good convention here.

Reply:
My thought process for separate files is it's more straightforward, since the place we link to changes over time, so you have to hunt around. Also this was easy to script, at least for Go, though I could see how more involved scripts would be a little different.

If you have a suggestion I'm happy to follow it.

Reply:
I merged as is, but realized afterward that one of the reasons for the symlinking is that it avoids actually having to run an identical script a dozen times. So I'm going to leave the recommendation as is, and we should try to use symlinks in the future.